src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -186,6 +186,11 @@ public void addGood(Good good) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public int indexOfGood(Good good) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
Why? Work in progress?

Reply:
```throw new AssertionError("This method should not be called.");``` is written for many of the ModelStub methods that are not used during the test.